test/odbc/main.cpp

```diff
@@ -1,8 +1,23 @@
 #include <gtest/gtest.h>
-#include "odbc_handler.h"
+#include <map>
+#include <string>
```

Comment:
nit: I think we can remove map and string here (should exist in "constants.h" which is included in "drivers.h". I believe it should be passed through multiple files but correct me if I'm wrong).

Reply:
Done.